### PR TITLE
Add Blog Post for ESG about UseGalaxy-it test instance

### PR DESCRIPTION
**Pull request by mtangaro**

### CODE REVIEW

File: content/news/2024-03-18-usegalaxy-it-starting/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "Enhancing the Galaxy Italian Community: Italy's National Galaxy Server"
+tease: ""
+hide_tease: true
+authors: "MA Tangaro (CNR), P Khmelevskaia (UniMi), F Zambelli (CNR and UniMi)"
+authors_structured:
+- github: mtangaro
+- github: Federico77z
+date: "2024-03-18"
+tags: [esg, esg-wp3]
+subsites: [all-eu, esg]
```
Comment (bgruening) [inline content/news/2024-03-18-usegalaxy-it-starting/index.md:11]:
```suggestion
subsites: [all-eu, esg, global]
```

**Comment by bgruening** [comment]
@mtangaro what is the status here?

**Comment by mtangaro** [comment]
> @mtangaro what is the status here?

I'm still debugging handlers config for jobs, as suggested nate with  py-spy

**Comment by bgruening** [comment]
Any update here?

**Comment by bgruening** [comment]
Can you please also rename the folder?

**Pull request merged**